images/win/Windows2016-Azure.json

```diff
@@ -107,9 +117,11 @@
         {
             "type": "powershell",
             "environment_vars": [
-                "ImageVersion={{user `image_version`}}"
+                "ImageVersion={{user `image_version`}}",
+                "TOOLSET_JSON_PATH={{user `toolset_json_path`}}"
             ],
             "scripts":[
+                "{{ template_dir }}/scripts/Installers/Install-PowerShellModules.ps1",
```

Comment:
Is it expected that we run `Install-PowerShellModules.ps1` before `Initialize-VM.ps1`?

Reply:
Yep, it's expected.